.github/workflows/create_release.yml

```diff
@@ -14,6 +14,31 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Generate developer build version
+        if: "startsWith(github.ref, 'refs/heads/')"
+        run: |
+          echo "build_version=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
```

Comment:
Nit: I think this can be on one line
run: echo "build_version=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV